docker-app/qfieldcloud/subscription/models.py

```diff
@@ -86,6 +86,22 @@ def get_or_create_default(cls) -> "Plan":
 
         return cast(Plan, result)
 
+    @classmethod
+    def get_plans_for_user(cls, user: User, user_type: User.Type) -> QuerySet["Plan"]:
+        """
+        Return all public plans of the given user_type, filtering out
```

Comment:
```suggestion
        Return all public plans of the given `user_type`, filtering out
```
Whenever we refer to something more programattic, we always try to put it in backticks.